extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/CacheProcessor.java

```diff
@@ -102,6 +114,7 @@ void validateCacheAnnotationsAndProduceCacheNames(CombinedIndexBuildItem combine
         for (DotName bindingName : INTERCEPTOR_BINDINGS) {
             for (AnnotationInstance binding : combinedIndex.getIndex().getAnnotations(bindingName)) {
                 throwables.addAll(validateInterceptorBindingTarget(binding, binding.target()));
+                findCacheKeyGenerator(binding, binding.target()).ifPresent(keyGenerators::add);
```

Comment:
Better avoid lambdas and method references here

Reply:
Does the "no-lambda policy" apply to build time code now? Last time I heard about it, it was only for run time code.

Reply:
It's runtime only

Reply:
Yes, runtime. I haven't noticed that this is build time code